src/DistributedLock.Tests/Tests/ApiTest.cs

```diff
@@ -57,9 +57,8 @@ public void TestProviderApisAreAvailable(AssemblyName assemblyName)
         {
             var providers = types.Where(t => !t.IsInterface && kvp.Key.IsAssignableFrom(t)).ToArray();
             var provided = types.Where(t => !t.IsInterface && kvp.Value.IsAssignableFrom(t)).ToArray();
-            CollectionAssert.AreEquivalent(
-                provided, 
-                providers.Select(t => t.GetMethods().Single(m => m.Name.StartsWith("Create") && kvp.Value.IsAssignableFrom(m.ReturnType)).ReturnType));
+            Assert.That(
+                providers.Select(t => t.GetMethods().Single(m => m.Name.StartsWith("Create") && kvp.Value.IsAssignableFrom(m.ReturnType)).ReturnType), Is.EquivalentTo(provided));
```

Comment:
I actually didn't realize there were changes like this; let me know if you want me to move `Is.EquivalentTo(provided)` to the next line.

Reply:
Fixed the whitespaces